woudc_data_registry/models.py

```diff
@@ -348,6 +354,7 @@ def __geo_interface__(self):
                 'station_name': self.station.station_name.name,
                 'data_class': self.dataset.data_class,
                 'dataset': self.dataset_id,
+                'contributor_name': self.contributor.name,
```

Comment:
As you can see in the relationships part of the class, there is no such thing as `self.contributor`. There is a `Deployment.contributor` though...

I think something is missing from your testing: you should definitely make sure `woudc-data-registry admin init` works before submitting this code. Oh, and `sync` as well. Those two would easily find issues like this.